destination/destination.go

```diff
@@ -61,6 +62,7 @@ func (d *Destination) Configure(ctx context.Context,
 
 // Open makes sure everything is prepared to receive records.
 func (d *Destination) Open(ctx context.Context) error {
+	// haris: out ofcuriosity, why not initialize it in NewDestination?
```

Comment:
Done.

Reply:
Hmm, can't see the change.

Reply:
It was moved to `Configure`, to keep all the destination init code together in one function

Reply:
It was moved to the sheets package, under `NewBatchReader`